server/services/netatmo/lib/commands/netatmo.connect.js

```diff
@@ -0,0 +1,85 @@
+const axios = require('axios');
+const querystring = require('querystring');
+const { ServiceNotConfiguredError } = require('../../../../utils/coreErrors');
+const { CONFIGURATION } = require('../constants');
+const { EVENTS, WEBSOCKET_MESSAGE_TYPES } = require('../../../../utils/constants');
+/**
+ * @description Connect.
+ * @example
+ * netatmo.connect();
+ */
+async function connect() {
+  // get value save by gladys front to connect to Netatmo
+
+  const netatmoClientId = await this.gladys.variable.getValue(CONFIGURATION.NETATMO_CLIENT_ID, this.serviceId);
+  const netatmoCientSecret = await this.gladys.variable.getValue(CONFIGURATION.NETATMO_CLIENT_SECRET, this.serviceId);
+  const netatmoUsername = await this.gladys.variable.getValue(CONFIGURATION.NETATMO_USERNAME, this.serviceId);
+  const netatmoPassword = await this.gladys.variable.getValue(CONFIGURATION.NETATMO_PASSWORD, this.serviceId);
+  const netatmoIsConnect = await this.gladys.variable.getValue(CONFIGURATION.NETATMO_IS_CONNECT, this.serviceId);
```

Comment:
still questioning wether is a good idea to use DB to store this. I think this is something that should be stored as attribute of the class like other services, not on the disk.

Reply:
And so where do you want to store this informations ? The raspberry just reboot and then you have to enter again username / password ?